src/app/protected/page.jsx

```diff
@@ -0,0 +1,35 @@
+import Image from "next/image";
+import ProtectedPage from "@/components/ProtectedPage";
+import { signOut, useSession } from "next-auth/react";
+import Link from "next/link";
+
+const Page = () => {
```

Comment:
ideally this should be in another file under components, or just combined with the below home function. rn its confusing as to which one is the actual page